src/api/app/models/binary_release.rb

```diff
@@ -126,6 +126,16 @@ def product_medium
     repository.product_medium.find_by(name: medium)
   end
 
+  # returns all package objects of docker containers where this binary is included
+  # usually just one, but people can build multiple docker containers with same name in one repo
+  def media_containers
+    # construct container name, there must be a better way ...
+    #               prefix + medium name excluding arch, Build number and suffix
+    containername = 'container:' << medium.gsub(/\.[^\.]*-[^-]*-Build[^-]*$/, '')
+    # must be in same repository since we got released here
+    Package.find(BinaryRelease.where(repository: repository, binary_name: containername).pluck(:release_package_id))
```

Comment:
`.find` will throw an exception if it goes wrong. Can this happen?

Reply:
dunno.... maybe when repository is invalid, but that should not be possible. An empty result should not lead to an exception...

Reply:
If you use `.find_by` it won't